modules/_labs/tabs/react/lib/TabList.tsx

```diff
@@ -1,16 +1,25 @@
 import * as React from 'react';
+
 import {spacing, commonColors} from '@workday/canvas-kit-react-core';
-import styled from '@emotion/styled';
-import {useTab} from './Tabs';
-import {TabProps} from './Tab';
+import {
+  composeHooks,
+  createComponent,
+  mergeProps,
+  styled,
+  StyledType,
+  useModelContext,
+} from '@workday/canvas-kit-react-common';
 
-type Tab = React.ReactElement<TabProps>;
+import {TabsModelContext} from './Tabs';
+import {useRovingFocus, orientationKeyMap} from './cursor/hooks';
+import {TabsModel} from './useTabsModel';
 
-export interface TabListProps extends React.HTMLAttributes<HTMLElement> {
-  /**
-   * A list of Tab components.
+export interface TabListProps {
+  children: React.ReactNode;
+  /** Optionally pass a model directly to this component. Default is to implicitly use the same
+   * model as the container component which uses React context. Only use this for advanced use-cases
    */
-  children: Tab | Tab[];
```

Comment:
We should not restrict children. It doesn't work very well anyways and it is perfectly valid to do the following:
```
<Tabs>
  <div> // Could add padding for the panels
    <Tabs.Panel>
```